libraries/AP_NavEKF3/AP_NavEKF3_Measurements.cpp

```diff
@@ -523,22 +526,22 @@ void NavEKF3_core::readGpsData()
     // limit update rate to avoid overflowing the FIFO buffer
     const AP_GPS &gps = AP::gps();
 
-    if (gps.last_message_time_ms() - lastTimeGpsReceived_ms > frontend->sensorIntervalMin_ms) {
-        if (gps.status() >= AP_GPS::GPS_OK_FIX_3D) {
+        if (gps.last_message_time_ms(selected_gps) - lastTimeGpsReceived_ms > frontend->sensorIntervalMin_ms) {
```

Comment:
I guess the "selected" gps is the same as the "active" gps?  It might be nice to keep the number of synonyms to a minimum because it makes the developer stop for a moment to wonder if they are the same or not.  i.e. It might be good to rename "selected_gps" to "active_gps".

Reply:
we might want to keep selected_gps so it is consistent with the other 2 (selected_baro, and selected_airspeed)